README-National-Cloud.md

```diff
@@ -0,0 +1,193 @@
+
+ ```Powershell
+ Connect-AzureAD -TenantId $tenantId -AzureEnvironmentName AzureUSGovernment
+ ```
+
+### Step 2:  Register the sample application with your Azure Active Directory tenant
+
+1. Sign in and register this sample in your Azure AD Tenant of your chosen National Cloud's [App registration portal](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-national-cloud#app-registration-endpoints) 
+2. On the top bar, click on your account and under the Directory list, choose the Active Directory tenant where you wish to register your application.
+3. Click on More Services in the left-hand nav, and choose Azure Active Directory.
+4. Click on App registrations and choose *Add*.
+5. Enter a friendly name for the application, for example `TodoListWebApp_MT` and select 'Web Application and/or Web API' as the Application Type. For the sign-on URL, enter the base URL for the sample, which is by default `https://localhost:44302/`.
+6. Click on *Create* to create the application.
+7. In the succeeding page, Find the *Application ID* value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
+8. Then click on *Settings*, and choose *Properties*.
+9. For the App ID URI, replace the guid in the generated URI 'https://<your_tenant_name>/<guid>', with the name of your service, for example, 'https://<your_tenant_name>/TodoListWebApp_MT' (replacing *<your_tenant_name>* with the name of your Azure AD tenant).
+10. From the Settings | Reply URLs page for your application, update the Reply URL for the application to be *https://localhost:44302/*
+11. From the Settings menu, choose Keys and add a key - select a key duration of either 1 year or 2 years. When you save this page, the key value will be displayed, copy, and save the value in a safe location - you will need this key later to configure the project in Visual Studio. This key value will not be displayed again, nor retrievable by any other means, so record it as soon as it is visible from the Azure Government portal.
+12. Configure Permissions for your application. To that extent, in the Settings menu, choose the 'Required permissions' section and then,
+   click on **Add**, then **Select an API**, and type `Microsoft Graph` in the textbox. Then, click on  **Select Permissions** and select **User.Read** and **User.Read.All**.
+
+### Step 3: Configure the sample to use your Azure AD tenant
+
+In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+Open the solution in Visual Studio to configure the projects
```

Comment:
Open the solution _dotnet-webapp-multitenant-oidc.sln_ in Visual Studio to configure the project  (no s).